apis/outlook/mail/code/go.mod

```diff
@@ -2,8 +2,11 @@ module github.com/gptscript-ai/tools/apis/outlook/mail/code
 
 go 1.23.0
 
+replace github.com/gptscript-ai/tools/apis/outlook/common => github.com/g-linville/tools/apis/outlook/common v0.0.0-20240903203431-d7bfd26dc70d
```

Comment:
This is pointing at my fork for now. Unfortunately I can't really change this until after I merge this PR. Should I open up a second PR to introduce the `common` package by itself first?

Reply:
I think it's fine to merge and immediately create another PR.

Reply:
any reason this common package has to be separate go module?

Reply:
I had separate Go modules for the manage and read tools, and want this code to be usable by both of them without one of them being the distinct "owner" of it